core/src/test/scala/unit/kafka/network/RequestChannelTest.scala

```diff
@@ -355,6 +357,71 @@ class RequestChannelTest {
     metrics = new Metrics(apis, config)
     testMetricsRequestSizeBucketDefault(metrics)
   }
+  @Test
+  def testHistogram(): Unit = {
+    var props = new Properties()
+    props.put(KafkaConfig.ZkConnectProp, "127.0.0.1:2181")
+    props.put(KafkaConfig.RequestMetricsTotalTimeBucketsProp, "0, 10, 20, 267")
+    var config = KafkaConfig.fromProps(props)
+    val requestMetrics = new RequestMetrics("RequestMetrics", config)
+    val boundaries = Array(0, 10, 30, 300)
```

Comment:
It's a bit confusing to see a mismatch between the RequestMetricsTotalTimeBucketsProp and the boundaries. I'd suggest making them consistent.

Reply:
updated as suggested